baybe/simulation/lookup.py

```diff
@@ -159,3 +139,41 @@ def _look_up_targets_from_dataframe(
 
     # Add the lookup values
     queries.loc[:, target_names] = np.asarray(all_match_vals)
+
+
+def label_columns(
+    input_labels: Sequence[str], output_labels: Sequence[str]
+) -> Callable:
```

Comment:
Specify `Callable`.

Reply:
You mean type hints? Just added them. Please resolve if this is what you were asking for